client/src/commands/run.ts

```diff
@@ -29,13 +29,31 @@ interface FoldingBlock {
 let running = false;
 
 function getCode(selected = false, uri?: Uri): string {
+  if (uri) {
+    console.log("run.ts::getCode() - uri.fsPath:", uri.fsPath && uri.fsPath);
+  }
   const editor = uri
     ? window.visibleTextEditors.find(
         (editor) => editor.document.uri.toString() === uri.toString(),
       )
     : window.activeTextEditor;
   const doc = editor?.document;
   let code = "";
+  let setCodeFile = "";
+  if (uri && uri.fsPath) {
+    setCodeFile =
+      "option set=SAS_EXECFILEPATH=%sysfunc(quote(" + uri.fsPath + "));\n";
```

Comment:
SAS_EXECFILEPATH is only available within the Enhanced Editor in Windows.
SAS Studio and SAS Enterprise Guide provides `&_SASPROGRAMFILE` macro variable for this requirement.
I would prefer to match Studio and EG as it's cross platform. What do you think?

Reply:
`&_SASPROGRAMFILE` is fine with me - will update and remove SAS_EXECFILEPATH

Reply:
Fine, I will update to use `&_SASPROGRAMFILE`